test/unit/src/extras/DataUtils.tests.js

```diff
@@ -1,6 +1,6 @@
 /* global QUnit */
 
-import { DataUtils } from '../../../../src/extras/DataUtils.js';
+import * as DataUtils from '../../../../src/extras/DataUtils.js';
```

Comment:
Because the import looks now different when directly importing from `src/extras/DataUtils.js` (not `three.module.js`), a note in the migration guide is required. I'll do that after the merge.